common/darkside.go

```diff
@@ -5,49 +5,176 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"errors"
+	"io"
+	"io/ioutil"
 	"os"
+	"os/exec"
 	"strconv"
+	"strings"
 	"time"
+
+	"github.com/zcash/lightwalletd/parser"
 )
 
-type DarksideZcashdState struct {
-	start_height       int
-	sapling_activation int
-	branch_id          string
-	chain_name         string
+type darksideState struct {
+	startHeight       int
+	saplingActivation int
+	branchID          string
+	chainName         string
 	// Should always be nonempty. Index 0 is the block at height start_height.
-	blocks                []string
-	incoming_transactions [][]byte
-	server_start          time.Time
+	blocks               [][]byte // full blocks, binary, as from zcashd getblock rpc
+	incomingTransactions [][]byte // full transactions, binary, zcashd getrawtransaction txid
+	serverStart          time.Time
```

Comment:
I started using vscode, and it has a built-in linter, so some of the changes here are just renaming it suggested, and other formatting with no functional effect.